### PR TITLE
tbi(fe-fpm-writer): Updates to improve consumption

### PR DESCRIPTION
**Pull request by IainSAP**
#650 

Adds 2 improvements:

- Template handling is now relative to the root (./dist) folder to help when bundling
- Pass the ui5 version through when creating FPM from FE writer

### CODE REVIEW

File: packages/fe-fpm-writer/src/templates.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { join } from 'path';
+/**
+ * Locates template files relative to the dist folder.
+ * This helps to locate templates when this module is bundled and the dir structure is flattened, maintaining the relative paths.
+ *
+ * @param relativeTemplatePath - optional, the path of the required template relative to the ./templates folder. If not specified the root templates folder is returned.
+ * @returns the path of the template specified or templates root folder
+ */
+export function getTemplatePath(relativeTemplatePath?: string): string {
+    return join(...[__dirname, '../templates'].concat(relativeTemplatePath || []));
```
Comment (tobiasqueck) [inline packages/fe-fpm-writer/src/templates.ts:10]: That line looks more complicated then it has to be. Wouldn't the snippet below do the same?
```
export function getTemplatePath(relativeTemplatePath: string = ''): string {
    return join(__dirname, '../templates', relativeTemplatePath);
```

**Review by devinea — APPROVED** [review]
Thanks @IainSAP 
Changes look good.

File: packages/fe-fpm-writer/test/unit/section.test.ts [hunk 2]
```diff
@@ -21,11 +21,11 @@ describe('CustomSection', () => {
             { version: 1.86, expected: join(root, 'section', '1.86') }
         ];
         test.each(testInput)('get root path of template', ({ version, expected }) => {
-            expect(getManifestRoot(root, version)).toEqual(expected);
+            expect(getManifestRoot(version)).toEqual(expected);
         });
         test('invalid version', () => {
             try {
-                getManifestRoot(root, 1.8);
+                getManifestRoot(1.8);
```
Comment (tobiasqueck) [inline packages/fe-fpm-writer/test/unit/section.test.ts:28]: It is not a problem but why did you change this test?

**Review by tobiasqueck — APPROVED** [review]
One question, otherwise, everything looks good. Clean code and and easy to read.

File: packages/fe-fpm-writer/test/unit/section.test.ts (hunk 2 above)
Comment (IainSAP) [reply packages/fe-fpm-writer/test/unit/section.test.ts:28]: `getManifestRoot`, which gets the manifest template root, no longer requires the templates root path to be passed in as it internally uses the new `getTemplatePath` function.

Comment (tobiasqueck) [reply packages/fe-fpm-writer/test/unit/section.test.ts:28]: Thanks!
